heterarchy.litcoffee

```diff
@@ -63,7 +63,18 @@ behaves like a class that would be have such a hierarchy.
             class Result extends generate tail linearization
                 __mro__: linearization
                 constructor: reparent next, @, next::constructor
+                # FILL UP MISSING CLASS ATTRIBUTES
+                # copy class attributes from `next` to `this` that only exist on `next`
                 copyOwn next, @
+                # ADJUST CLASS METHOD (so the MRO is used)
+                # copy class methods from `next` to `this` that are implemented by `next`
+                # note: this is basically the same as copyOwn but ignoring `if not to.hasOwnProperty key`
+                # This is due to the fact that when inheriting all class attributes are copied to `this` by CoffeeScript.
```

Comment:
Yep, python-like 😄 Depending on how complicated squashing is, that could be one of those things you could do after the merge (otherwise it would just create another commit that I'd need to handle/squash)